tools/install_protoc.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+if [[ "$(uname)"=="Darwin" || "$(expr substr $(uname -s) 1 10)"=="MINGW32_NT" ]];then
```

Comment:
I saw there is a binary for MacOS too, why not support MacOS here as well?

Reply:
I think he is setting CI? Which only include Ubuntu for now? We should set up the MacOS for CI testing too.

Reply:
In this PR, he's using a third-party plugin instead of this bash to install protoc

Reply:
> I think he is setting CI? Which only include Ubuntu for now? We should set up the MacOS for CI testing too.

Locally, when you run `make deps` to setup the dev env, this is executed and if you didn't install protoc correctly, it fails.

Reply:
I tried, I have to install the protoc manually.